etl/steps/data/garden/oecd/2025-03-26/government_at_a_glance.meta.yml

```diff
@@ -0,0 +1,625 @@
+
+  description_from_producer_tax_revenues: >-
+    Data on revenues are computed from the OECD
+    National Accounts Statistics (database), which are based
+    on the System of National Accounts (SNA). the 2008
+    SNA framework has been implemented by all OECD
+    countries. revenues include taxes
+    (e.g. on consumption, income, wealth, property and
+    capital), net social contributions (i.e. contributions for
+    pensions, health and social security after deduction
+    of social insurance scheme service charges, where
+    applicable), sales of goods and services (e.g. market
+    output of government establishments, entrance
+    fees), and grants and other sources (e.g. current and
+    capital grants, property income, and subsidies). These
+    aggregates were constructed using sub-account
+    items. The definitions of tax
+    revenues differ between SNA and OECD Revenue
+    Statistics, especially regarding compulsory social
+    security contributions. In SNA, taxes are mandatory
+    unrequited payments, in cash or in kind, made by
+    institutional units to the government. Net social
+    contributions are actual or imputed payments to
+    social insurance schemes to make provision for
+    social benefits to be paid. These may be compulsory
+    or voluntary and funded or unfunded. OECD
+    Revenue Statistics treat compulsory social security
+    contributions as taxes, whereas the SNA considers
+    them net social contributions because the receipt of
+    social security benefits depends, in most countries,
+    upon appropriate contributions having been made,
+    even though the size of the benefit is not necessarily
+    related to the amount of the contributions.
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  title: OECD Government at a Glance
+  update_period_days: 365
+
+
+tables:
+  government_at_a_glance:
+    variables:
+      government_expenditure:
+        title: Government expenditure ({definitions.unit_human_readable})
+        unit: "{definitions.unit}"
+        short_unit: "{definitions.short_unit}"
+        description_short: Total expenditure from central, state and local governments and social security funds. {definitions.short_description_unit}
+        description_key:
+          - "{definitions.key_government_expenditure}"
+          - "{definitions.key_general_government}"
+        description_from_producer: "{definitions.description_from_producer_expenditure}"
+        display:
+          name: Government expenditure ({definitions.unit_human_readable})
+          numDecimalPlaces: "{definitions.num_decimal_places}"
+          <<: *common_display
```

Comment:
Are all `display.name`s identical to the title? If so, you could simply remove the definitions of `display.name`s.
Otherwise, if you define a display name that is different to the title, then you should also include a `presentation.title_public`.

Reply:
Thanks, I will keep them, because they are different in the case of the indicators using economic_transaction.

Regarding presentation.title_public, I am currently inclined to not use it, because in its absence, the data pages now show the title of the chart and I avoid mismatch there. We should in fact change or delete that warning that shows up in the terminal

Reply:
The field `display.name` is [intended](https://docs.owid.io/projects/etl/architecture/metadata/reference/#variabledisplayname) to be used to specify the title in the legend of a chart (for backwards compatibility, `display.name` is also used to replace the title, but this is not its intended use).
It seems you are using `display.name` for the title of the chart, in which case you should instead use `presentation.title_public`. Or am I missing something?

Reply:
The warning is there because, if you define `display.name` as a very short name, e.g. "Expenditure", just for a specific legend in a chart, then this short and uninformative title will also be displayed as the title of the data page, which would be bad.

Reply:
No, I am not intending to use it as the title of a chart. But what is happening now is not adding a `presentation.title_public`, the data page will use the title I manually set in the chart. This way I avoid mismatches between the manually edited title and title_public. At least in my charts I never have the titles linked to these fields.